packages/http-client-java/generator/http-client-generator-test/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "type": "module",
   "scripts": {
+    "clean": "rimraf ./node_modules/@typespec/http-client-java ./package-lock.json",
```

Comment:
Here `npm run clean` would delete the related file, so that the next `npm install` would install latest http-client-java package to test module.